packages/core/src/lib/wrappers/svg2img.ts

```diff
@@ -0,0 +1,30 @@
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const _svg2img = require('svg2img');
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export type Callback = (err: any, buffer: any) => any;
+
+export enum Format {
```

Comment:
Do you mind adding a comment somewhere in this class that they're only reason we're doing this is that we couldn't get their types to work, and that if we do in the future, we can remove most of this.